libvips/iofuncs/header.c

```diff
@@ -367,7 +367,7 @@ meta_init( VipsImage *im )
 }
 
 /**
- * vips_image_get_width: (method)
```

Comment:
Have you removed this because width is also available as a property? Or have I misunderstood the method annotation (quite likely).